include/wx/filesys.h

```diff
@@ -16,6 +16,7 @@
 #include "wx/stream.h"
 #include "wx/datetime.h"
 #include "wx/filename.h"
+#include "wx/list.h"
```

Comment:
Unrelated, but I think we could change type of `m_Handlers` to `std::list<>` without breaking compatibility: nobody is supposed to use it directly anyhow.